ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/services/CommandCompletionService.scala

```diff
@@ -37,6 +42,10 @@ class CommandCompletionService(
       ledgerId: String,
       config: WorkflowConfig.StreamConfig.CompletionsStreamConfig,
   ): CompletionStreamRequest = {
+    assert(
```

Comment:
fyi @mziolekda relating to https://github.com/digital-asset/daml/pull/12861#issuecomment-1034786603
With user-based authorization enabled, and given that benchtool runs as (hardcoded) `benchtool` user, the `config.applicationId` has to be `benchtool` as otherwise we get something like permission_denied/unathenticated error. That's the limitation of the user-auth support I'm adding in this PR, but I think it's not worth to invest in making it more flexible at this point.

With user-based authorization disabled `config.applicationId` can be anything we want.